mlflow/server/__init__.py

```diff
@@ -50,6 +52,14 @@
         os.makedirs(prometheus_metrics_path)
     activate_prometheus_exporter(app)
 
+if os.getenv(ENABLE_STATISTICS_ENV_VAR, "false") == "true":
+    from mlflow.server.statistics_collector import MlflowStatisticsCollector
+
+    statistics_update_interval = float(os.getenv(STATISTICS_UPDATE_INTERVAL_ENV_VAR, "3600"))
```

Comment:
Is there a lower bound to this that shouldn't be allowed? If a user sets this to, say, 1 second, what is the impact on the monitoring table?
This setting can also be migrated to ``environment_variables``

Reply:
I will ensure that.